Weekly Changelog 2023-08-28 to 2023-09-04 (#68002)
CleverRaven/Cataclysm-DDA: data/changelog.txt

Adds a chance for small, personal photographs to spawn within people’s wallets and creates a starting pool of 34 snippets from which to pull.
Adds curved road bends, rotaries and rest stops/laybys
JSONify Defense Mode

> Add coconut bowls and coconut water
> Cowboy & Rancher professions spawns with a long rope for animals.
> Add a new mission to the exodii, as described by #59905
> Add boats to river banks
> Added more vehicles into the game
> Office skyscraper revival

Reviewer: ```suggestion
Adds a new mission to the Exodii
Add boats to river banks
```

Detailed information for stats on character creation menu
Larger and more I18n-friendly safemode UI
Enable/disable showing several non-player-related messages in the log

> Merge Object Creator windows into a tabbed view
> Separate Proficiency and Skill training speed option

Reviewer: ```suggestion
```

[MoM] Add Enervation damage type, apply it to Eater and feral vitakinetics
[Magiclysm] Add more than a dozen spells to magiclysm
[Magiclysm] Add fantasy species starting option

> Add two more playable fantasy species for Magiclysm
> Add even more Magiclysm spells (+ lab special)
> Add RNG to crystallized mana spawns
> Rework Nature's Bow to require ammo, add spell to create ammo
> [DinoMod] more interesting behavior part 1
> Add ancient stone circle (with translocate altar)

Reviewer: ```suggestion
[Magiclysm] Add two more playable fantasy species for Magiclysm
```

Limit the times assassins can try to kill you.
Increase plastic variety, adjust plastics to be more realistic in terms of protection
Characters start with basic skills from their previous life

> fewer gas zombies

Reviewer: ```suggestion
```

Fix appliance power drain display right after plugging in/unplugging device
Fix items and furniture being deleted from grappling hook usage
Randomly generated characters are now aged appropriately to their profession

> Exploding zombie cows with landmines
> Nerf lift thrust of rotorcraft by 6 orders of magnitude
> Fix Ravenfolk mutation category name
> Remove misplaced carpet from steel mill
> Fix rotorcraft debug message
> Adjust the storage volume and fuel type of the backup generator.

Reviewer: ```suggestion
```

Optimize eoc processing and other fixes to speed up waiting near many npcs
Optimize pocket overflow function
Speed up new character screen, particularly when many recipes are known

> Improve turn performance by caching default input ctx
> Revert: Improve turn performance by caching default input ctx

Reviewer: ```suggestion
```

New item categories for martial arts manuals and traps
Update all active items to new tick action system and remove old system
Add ability to merge appliance into grid

> Distinguish sieving and straining in recipes

Reviewer: ```suggestion
```
